guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>"The <code>enrich()</code> method returns a Spatiallly Enabled Data Frame that can be saved as a new Item in your GIS and used for analysis or visualization on a map." </p><ul><li>Mohit, can you please rephrase this line? I am a little confused at how the SeDF that can be saved as item??</li></ul><p> </p><p>"However, if you would like to enrich an existing FeatureLayer <s>by adding new data</s>, then use the <code>enrich_layer()</code> method from the (arcgis.)features module. The result will be a new layer of input features that include<s>s</s> enriched data."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
rephrased the line. please see if it makes sense now.